app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -103,6 +245,226 @@
     margin: 1.5rem 0;
   }
 
+<<<<<<< HEAD
+  /* Step 1 */
+  .getting-started {
+
+    .crop-name-and-description {
+      margin-bottom: 1.5rem;
+    }
+
+    .crop-name {
+      font-size: 1.8rem;
+      /* account for cancel button */
+
+      .common-name {
+        padding-right: 1.5rem;
+      }
+
+      .binominal-name {
+        color: #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.